AccountService.js

```diff
@@ -40,4 +40,72 @@ export class AccountService {
     });
     return response.data;
   }
+
+  /**
+   * @method accounts
+   * @param {Object} config
+   * @param {string} config.url the base url
+   * @param {string} config.email the user's email
+   * @param {string} config.after an account's DID
+   * @param {number} config.limit how many accounts to return
+   * @return {Array} data
+   * @description calls on the adminr route and returns all
+   * accounts that match the email passed in.
+   * the cursor should be an account's DID
+   * and the limit is number.
+   */
+  async accounts({url = this.config.urls.base, email, after, limit}) {
+    const response = await axios.get(url, {
+      params: {email, after, limit},
+      headers: {'Accept': 'application/ld+json, application/json'}
+    });
+    return response.data;
+  }
+
+  /**
+   * @method update
+   * @param {Object} config
+   * @param {string} config.url
+   * @param {string} config.id
+   * @param {number} config.sequence
+   * @param {Array<Object>} config.patch
+   * @return {Void}
+   * @description id is an account's DID,
+   * sequence is an integer
+   * patch is an array of json-patches
+   */
+  async update({url = this.config.urls.base, id, sequence, patch}) {
+    await axios.patch(`${url}/${id}`,
+      {sequence, patch},
+      {headers: {'Accept': 'application/ld+json, application/json'}});
+  }
+
+  /**
+   * @method setStatus
+   * @param {Object} config
+   * @param {string} config.url
+   * @param {string} config.id
+   * @param {string} config.status
+   * @return {Void}
+   * @description taken an id and a status string
+   * patches an account's status
+  */
+  async setStatus({url = this.config.urls.base, id, status}) {
+    await axios.post(`${url}/${id}/status`,
+      {status},
+      {headers: {'Accept': 'application/ld+json, application/json'}});
+  }
+  /**
```

Comment:
Anyone care about whether or not there is a newline here?

Reply:
I'd like one.

Reply:
the style @dlongley wanted is to only introduce a line break when a function's params are longer than the max line. This why consistent and func-parens was removed from the rules.

Reply:
I think this is about a line break before the new comment section. Which I'd also like to see.